ocs_ci/utility/utils.py

```diff
@@ -4739,3 +4739,33 @@ def exec_nb_db_query(query):
         output = output[2:-1]
 
     return output
+
+
+def get_role_arn_from_sub():
+    """
+    Get the RoleARN from the OCS subscription
+
+    Returns:
+        role_arn (str): Role ARN used for ODF deployment
+
+    Raises:
+        ClusterNotInSTSModeException (Exception) if cluster
+        not in STS mode
+
+    """
+    from ocs_ci.ocs.ocp import OCP
```

Comment:
Please Move this import at the beginning of file

Reply:
This is intentionally imported here to avoid circular import